lifecycle-management/ahk-lifecycle-management/ahk-lifecycle-management.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <RootNamespace>Ahk.Lifecycle.Management</RootNamespace>
+  </PropertyGroup>
```

Comment:
Ha megnézed egy másik function app csproját, itt még van pár beállítás. Azok kellenek ide is.

Reply:
done